lib/stripe/payment_methods/bank_account.ex

```diff
@@ -21,7 +21,9 @@ defmodule Stripe.BankAccount do
           default_for_currency: boolean | nil,
           fingerprint: String.t() | nil,
           last4: String.t(),
-          metadata: Stripe.Types.metadata() | nil,
+          metadata: %{
+            optional(String.t()) => String.t()
+          } | nil,
```

Comment:
What's the reasoning for replacing this? I'm not saying it's wrong and I vaguely recall a conversation with someone about it, but it looks like `Stripe.Types.metadata` is defined as exactly this here, so it sounds reasonable to leave it as is.

Reply:
I looked at the `coupon` PR.  But now looking in `types.ex`, this is the exact same thing.  Will revert 👍